metricbeat/module/system/raid/raid_test.go

```diff
@@ -0,0 +1,25 @@
+// +build linux
+
+package raid
+
+import (
+	"testing"
+
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
+
```

Comment:
Consider removing this newline for me 😄 . (BTW I have a tool for this at [nonewlines](https://github.com/andrewkroh/go-nonewlines))

Reply:
@andrewkroh You should add this tool to our `make fmt` part so CI can bug us in the future instead of requiring the krohcompiler. Like this you can also go on vacation :-D

Reply:
It will be part of gofmt in Go 1.10. I'm just trying to prepare you in advance.

Reply:
I need to check if it was you that contributed it to Go 1.10 :-)

Reply:
BTW: Probably worth adding the script it will take some time until we are on 1.10 (lots of newlines to come until then)